### ISSUE TITLE
Game crash upon attempt to save

### ISSUE THREAD

**Issue by benkorpella** [CammiePone/Hookshot#17]
My game crashed when I tried to save my game after I /summoned a hookshot entity. I have attached two crash reports.

Steps to reproduce: 
1. /summon hookshot:hookshot
2. Don't give it an "Owner" nbt tag
3. Save the game

This isn't a high priority, as it requires you to specifically summon a hookshot. So, just don't do that and you'll be fine.

[crash-2021-08-19_18.32.00-server.txt](https://github.com/CammiePone/Hookshot/files/7018832/crash-2021-08-19_18.32.00-server.txt)
[crash-2021-08-19_21.44.27-server.txt](https://github.com/CammiePone/Hookshot/files/7018833/crash-2021-08-19_21.44.27-server.txt)

**Comment by benkorpella** [comment]
Upon further search, this might be a duplicate of #13. Although, it is still happening on r18.

**Comment by CammiePone** [comment]
Nah, the /summon command is something I would expect to cause problems because I never safely check if there is an owner, since under no normal circumstance should there not be one

**Comment by MJCaister** [comment]
I'm getting this issue with a player on my server. No cheats are enabled.

[crash-2021-09-30_10.34.13-server.txt](https://github.com/CammiePone/Hookshot/files/7258849/crash-2021-09-30_10.34.13-server.txt)
[crash-2021-09-30_10.01.35-server.txt](https://github.com/CammiePone/Hookshot/files/7258850/crash-2021-09-30_10.01.35-server.txt)
